docs/DafnyRef/README.md

```diff
@@ -44,7 +44,7 @@ Tools providing syntax coloring for programming language source typically
 require a PL-specific regular-expression based definition of the language
 syntax. Unfortunately there are many such tools.
 
-In the RM maarkdown sources, text bracketed by '```dafny' and '```' will have
+In the RM markdown sources, text bracketed by '```dafny' and '```' will have
```

Comment:
On this line we should also fix the backticks:

![image](https://user-images.githubusercontent.com/8181630/155030677-fdb8e536-c222-4a84-8bdf-b69334e96ade.png)